google/resource_cloudfunctions_function.go

```diff
@@ -129,7 +129,7 @@ func resourceCloudFunctionsFunction() *schema.Resource {
 			"source_archive_object": {
 				Type:     schema.TypeString,
 				Required: true,
-				ForceNew: true,
+				ForceNew: false,
```

Comment:
Because the empty value for a boolean is `false`, we can just delete this line entirely.